### PR TITLE
fix: Whole card clickable

### LINKED ISSUE #8985: Whole card clickable
currently the card is clickable only on the img, and the title 

- the whole area, apart from the collection name should be clickable and redirect user to the item

image

- apply this to all cards please 

also cursor -> pointer

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

  👇 __ Let's make a quick check before the contribution.

  ## PR Type

  - [x] Feature

  ## Needs Design check

  - @exezbcz please review

  ## Context

  - [x] Closes #8985

  #### Did your issue had any of the "$" label on it?

  - [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)

  ## Screenshot 📸

  - [x] My fix has changed UI
image

  ## Copilot Summary
  copilot:summary

  copilot:poem

### CODE REVIEW

**Comment by exezbcz** [comment]
Nice screenshot XD, will check soon

File: components/carousel/module/CarouselAgnostic.vue [hunk 1]
```diff
@@ -32,17 +35,20 @@
 <script lang="ts" setup>
 import CarouselMedia from './CarouselMedia.vue'
 import CarouselInfo from './CarouselInfo.vue'
+const NuxtLink = resolveComponent('NuxtLink')
```
Comment (preschian) [inline components/carousel/module/CarouselAgnostic.vue:38]: safe to remove this line?

**Review by preschian — APPROVED** [review]
otherwise lgtm

File: components/carousel/module/CarouselAgnostic.vue (hunk 1 above)
Comment (Jarsen136) [reply components/carousel/module/CarouselAgnostic.vue]: Yes, it works

**Comment by yangwao** [comment]
thanks!
pay 20 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $20 @ 7.59 USD/DOT ~ 2.635 $DOT
🧗 [16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z](https://polkadot.subscan.io/account/16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)
🔗 [0x2408f23478b5c89ea8cb1cfd40efcd592ac02afca6167dd060419bfcc5455ae7](https://polkadot.subscan.io/extrinsic/0x2408f23478b5c89ea8cb1cfd40efcd592ac02afca6167dd060419bfcc5455ae7)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**